### DISCUSSION TITLE
Generate model with singular name table

### DISCUSSION THREAD

**Discussion by hoanbv** [krlove/eloquent-model-generator#99] [Q&A]
i have a table name is "user_type" , how can I generate a  model with singular name table?
when i run 
`php artisan krlove:generate:model UserType --output-path=models --namespace=models`
i receied an error 
` Krlove\EloquentModelGenerator\Exception\GeneratorException  : Table user_types does not exist`

**Comment by Omar-Talbi** [comment]
@hoanbv did you find any solution for those sigular tables names ? i have the same problem when i try `php artisan krlove:generate:models` and in my database i have so many signular tables names
